docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -809,10 +809,12 @@ these pieces of data from it. It then performs the following actions:
   type="tip"
   title="Note"
 >
+Trusted Clusters work only in one direction. In the example above, users from
+the leaf cluster cannot see or connect to Nodes in the root cluster.
 
-  Trusted Clusters work only in one direction. In the example above, users from
-  the leaf cluster cannot see or connect to Nodes in the root cluster.
-
+The Teleport root cluster's web UI will not populate available OS users for
```

Comment:
Should we capitalize "web UI" throughout the docs? Right now the balance is: 

```
 $ grep -RhEoi "web UI" docs/pages | sort | uniq -c
 166 Web UI
  43 web UI
```

Reply:
```suggestion
The Teleport root cluster's Web UI will not populate available OS users for
```